<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogTag</class>
 <widget class="QWidget" name="dialogTag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="styleSheet">
      <string notr="true">QToolButton
{
	border-style: flat;
	background: transparent;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QToolButton" name="BtnRec">
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/矩形.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/矩形选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnCircle">
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/椭圆.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/椭圆选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnPolygon">
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/四边形.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/四边形选中.png);
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>201</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton
{
	border-style: flat;
	background: transparent;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QToolButton" name="BtnRed">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/红色.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/红色选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnOriange">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/橙色.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/橙色选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnYellow">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/黄色.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/黄色选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnGreen">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/绿色.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/绿色选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnBlue">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/蓝色.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/蓝色选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnDarkBlue">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/深蓝.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/深蓝选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="BtnPurple">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton{
 
	image: url(:/new/prefix1/image/紫色.png);
}
QToolButton:hover{
    
	image: url(:/new/prefix1/image/紫色选中.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
